.github/workflows/ci.yml

```diff
@@ -180,12 +192,20 @@ jobs:
         run: ci/github/codecov.sh "setup"
 
       - name: Run tests
+        if: '!matrix.coverity'
         run: ci/build.sh
 
       - name: Upload coverage
         if: matrix.coverage
         run: ci/codecov.sh "upload"
 
+      - name: Run coverity
+        if: matrix.coverity && github.event_name == 'push'
```

Comment:
@Flamefire You didn't provide enough guidance (I can't tell what you prefer).  Do you want an extra job or do you want to keep it as is?  If you are okay keeping it as it is, I would avoid changing the style issue above and approve, so we can get this in, because it's painful to do another cycle on this.  If you want another job, that's fine, I just didn't like the code duplication but that's what GHA forces us to do.

Reply:
It would be okay to run on feature branches, I think.

Reply:
I'll plan to make it a separate step.  You are correct that it shouldn't have too many steps.  This way the job itself can be disabled on pull requests.

Reply:
Let's get the multiarch one in then I can work on another job for this.

Reply:
Alright. I committed the 2 suggested changes so this is at least consistent and will merge this now. Thanks for the patience.